pkgs/servers/demoit/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv
+, buildGoPackage
+, fetchFromGitHub
+}:
+
+buildGoPackage rec {
+  name = "demoit";
```

Comment:
The name should include a version. If upstream does not release versions at all, then use the date in YYYYMMDD
format.

Reply:
Sorry, I should have been more specific:
* the date should be the date of the commit you package, so as to be meaningful
* the date should include the leading 0 of months and days, so as to be increasing across updates
here: 20190211

Reply:
Now it should be ok